.test-infra/jenkins/job_beam_Inventory.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// These jobs list details about each beam runner, to clarify what software
+// is on each machine.
+def nums = 1..8
+nums.each {
+  def machine = "beam${it}"
+  job("beam_Inventory_${machine}") {
+    description("Run inventory on ${machine}")
+
+    // Set common parameters.
+    common_job_properties.setTopLevelMainJobProperties(delegate)
+
+    // Sets that this is a PostCommit job.
```

Comment:
PostCommit and Cron seem like they shouldn't really be in the same category, if we want to run inventory regularly, independently of changes (which seems appropriate)

(I recognize that this isn't introduced here and shouldn't be a blocker, but it's worth noting)

Reply:
Fixed. Added a method in common, setCronJob and called that instead.